src/bundle/Resources/public/js/scripts/admin.anchor.sections.navigation.js

```diff
@@ -0,0 +1,156 @@
+(function (global, doc) {
+    if (!doc.querySelector('.ibexa-navigation-menu')) {
+        return;
+    }
+
+    const SECTION_ADJUST_MARGIN_TOP = 20;
+    const formContainerNode = doc.querySelector('.ibexa-edit-content');
+    const getSecondarySectionActiveItems = () => {
+        const secondarySectionItems = formContainerNode.querySelectorAll(
+            '.ibexa-edit-content__primary-section--active .ibexa-edit-content__secondary-section',
+        );
+
+        return [...secondarySectionItems];
+    };
+    let currentlyVisibleSections = getSecondarySectionActiveItems();
+    const fitSecondarySections = () => {
+        const primarySection = doc.querySelector('.ibexa-edit-content__primary-section--active');
+        const contentColumn = doc.querySelector('.ibexa-main-container__content-column');
+        const firstSection = primarySection.querySelector('.ibexa-edit-content__secondary-section:first-child');
+        const lastSection = primarySection.querySelector('.ibexa-edit-content__secondary-section:last-child');
```

Comment:
This can be confusing because `:first-child` and `:last-child` _"ignore"_ the class. This will return `null` in case `.ibexa-edit-content__secondary-section` is not first/last element, but it is assumed below that `firstSection` will never be null, so:
```suggestion
        const firstSection = primarySection.firstElementChild;
        const lastSection = primarySection.lastElementChild;
```